robot_skills/src/robot_skills/base.py

```diff
@@ -143,6 +143,9 @@ def computePathLength(self, path):
                 distance += math.sqrt( dx*dx + dy*dy)
         return distance
 
+    def reset(self):
```

Comment:
Already in RobotPart. Plus should return a bool.

Reply:
should be `self._action_client.cancel_all_goals()`; return true

Reply:
Nietes! GlobalPlanner has 2 service clients only, those can't be cancelled.